ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__aws-4.19-okd-x86.yaml

```diff
@@ -0,0 +1,114 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: okd
+      stream: okd-scos
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: compact-cp-3nodes
+  steps:
+    allow_skip_on_success: true
```

Comment:
lets flip this to collect.

Reply:
How this is done?
Is there a env variable or some step?
Can you please point me to any examples?
As part of the job I see the following steps:
- gather-audit-logs
- gather-aws-console
- gather-extra/
- gather-must-gather/

Reply:
It was ran because that job failed.